intelmq/lib/exceptions.py

```diff
@@ -15,8 +15,16 @@ def __init__(self, message):
 
 class InvalidArgument(IntelMQException):
 
-    def __init__(self, argument):
-        message = "argument %s is invalid" % repr(argument)
+    def __init__(self, argument, got=None, expected=None, docs=None):
```

Comment:
It was very handy to have more informations about the actual wrong value (not just the variable name) when working on the pipeline tests.